.github/CODEOWNERS

```diff
@@ -0,0 +1,68 @@
+# This list auto requests reviews from the specified org members
+# when a PR that modifies the file in question is opened
+# This list is alphabetized by User -> Filename KEEP IT THAT WAY
+
+# ChangelingRain
+
+/code/__DEFINES/clockcult.dm @ChangelingRain
+/code/datums/antagonists/datum_clockcult.dm @ChangelingRain
+/code/game/gamemodes/clock_cult/* @ChangelingRain
+
+# Cyberboss
+
+/code/__DATASTRUCTURES/globals.dm @Cyberboss
+/code/__DEFINES/components.dm @Cyberboss
+/code/__DEFINES/server_tools.dm @Cyberboss
+/code/controllers/subsystem/atoms.dm @Cyberboss
+/code/controllers/subsystem/mapping.dm @Cyberboss
+/code/controllers/globals.dm @Cyberboss
+/code/datums/helper_datums/getrev.dm @Cyberboss
+/code/datums/components/* @Cyberboss
+/code/datums/map_config.dm @Cyberboss
+/code/datums/forced_movement.dm @Cyberboss
+/code/datums/holocall.dm @Cyberboss
+/code/modules/admin/verbs/adminhelp.dm @Cyberboss
+/code/modules/admin/verbs/adminpm.dm @Cyberboss
+/code/modules/server_tools/* @Cyberboss
+/code/modules/mapping/* @Cyberboss
+
+# Iamgoofball
+
+/code/modules/GOAP/* @Iamgoofball

```

Comment:
considering I wrote the GOAP framework, you should put me on it too (and really only put goof on the actual examples like GOAP medibots, GOAP use cases etc. cos he made those)